src/area.jl

```diff
@@ -5,7 +5,7 @@ function area(poly::T) where T
         p2 = poly[i+1]
         a += p1[1]*p2[2]-p2[1]*p1[2]
     end
-    return a/2
+    return abs(a)/2
```

Comment:
Can we put this behind a flag? For some of my applications signed area is important. Also this would be breaking, so maybe leave the default signed and make it an option? 

```suggestion
    return signed ? a/2 : abs(a)/2
```

Reply:
I've never heard of a signed area. What is it used for?

Reply:
You can use it to check winding order + area at the same time.

Reply:
Ah interesting. Perhaps that's okay then and should just be documented? Is `area(points, true)` better than `abs(area(points))` ?

Reply:
Maybe it would be nicer to just add an `unsignedarea` to avoid the branch?